F2/genotyping.md

```diff
@@ -0,0 +1,142 @@
+
+---
+# Serotyping
+
+There are over 100 known _S. pneumoniae_ serotypes and the gold standard for serotyping is the Quellung reaction. Serotype can also be confidently predicted directly from short read data using _in silico_ tools including SeroBA, which is a _k_-mer based method that accurately detects the _cps_ locus even at low coverage and is scalable to large datasets [(Epping et al., 2018)](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000186). It is important to note that the results from _in silico_ prediction sometimes do not match the phenotypic result, and should not be used for clinical purposes. Pathogenwatch integrates the SeroBA tool into an easy to use platform which we will introduce in the practical section of the module.
+
+---
+# Antimicrobial resistance (AMR) prediction
+
+Genotypic resistance can be predicted _in silico_ from whole genome sequencing (WGS) data. Successful _in silico_ prediction of AMR phenotype from WGS data is dependent upon sequence quality and having a comprehensive database of antimicrobial resistance genes (ARGs) and mutations conferring AMR.
+
+There are publicly available databases of AMR genes and point mutations, with varying levels of comprehensiveness, curation and regularity of updates. The method and database used influence the sensitivity and specificity of detection of ARGs and point mutations, and should be considered when comparing studies and outputs of surveillance programmes ([Doyle et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32048983/)). Multiple studies have shown high concordance between AMR phenotype and genotype, including for _S. pneumoniae_, though specificity and sensitivity varies between antimicrobials ([Metcalf et al., 2016](https://www.sciencedirect.com/science/article/pii/S1198743X16303020); [Neuert et al., 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880904/); [Zankari et al., 2012](https://academic.oup.com/jac/article/68/4/771/705760?login=false)).
+
+It is important to remember that current _in silico _AMR prediction is not suitable for clinical purposes; they do not detect novel AMR mechanisms or variants with low homology to other genes in the database used, and require investigation with approaches such as genome wide association studies . Additionally, an ARG may be present but not expressed, resulting in non-concordance between genotype and phenotype ([Feldgarden et al., 2019](https://pubmed.ncbi.nlm.nih.gov/31427293/)).
```

Comment:
I think the "_"  after silico needs to be moved left by one space, so that "in silico" is italicised (currently seen as _in silico _AMR)